scripts/speakers.js

```diff
@@ -0,0 +1,85 @@
+const speakers = [
+  {
+    name: 'Elon Musk',
+    job: 'Founder',
+    photo: './assets/images/elon.jpg',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+    Labore, necessitatibus.`,
+  },
```

Comment:
- [ ] The same thing here, there are differences in how the `Featured Speakers` sections of yours and original are placed. Yours is placed vertically while the original is horizontal. Let's fix it, too 👇 
- Original :
![Screenshot (166)](https://user-images.githubusercontent.com/67087664/152135033-60fe89de-accc-42d6-a904-c7da80877f32.png)
- Your design below:
![Screenshot (165)](https://user-images.githubusercontent.com/67087664/152135098-9f3f2903-c920-40ce-8d9b-02bd334f1efc.png)
Remember this is for a desktop version.